### PR TITLE
Adjust syntax for ParentBased sampler to distinguish inputs from output

### LINKED ISSUE #2216: Parent-based Sampling docs
On https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/sdk.md:

```
  * Remote parent (`SpanContext.IsRemote() == true`) with `SampledFlag` equals `true`
  * Remote parent (`SpanContext.IsRemote() == true`) with `SampledFlag` equals `false`
  * Local parent (`SpanContext.IsRemote() == false`) with `SampledFlag` equals `true`
  * Local parent (`SpanContext.IsRemote() == false`) with `SampledFlag` equals `false`
```

Something's amiss here. Lines 1&2 have the same input, with a different output... And the same problem exists between the 3rd & 4th lines.

### PR DESCRIPTION
**Pull request by arminru**
Fixes #2216.

### CODE REVIEW

**Review by yurishkuro — APPROVED** [review]

File: specification/trace/sdk.md [hunk 1]
```diff
@@ -328,10 +328,10 @@ of the `TraceID`. [#1413](https://github.com/open-telemetry/opentelemetry-specif
 * This is a composite sampler. `ParentBased` helps distinguish between the
 following cases:
   * No parent (root span).
-  * Remote parent (`SpanContext.IsRemote() == true`) with `SampledFlag` equals `true`
-  * Remote parent (`SpanContext.IsRemote() == true`) with `SampledFlag` equals `false`
-  * Local parent (`SpanContext.IsRemote() == false`) with `SampledFlag` equals `true`
-  * Local parent (`SpanContext.IsRemote() == false`) with `SampledFlag` equals `false`
+  * Remote parent (`SpanContext.IsRemote() == true`) with `SampledFlag == true`
```
Comment (anuraaga) [inline specification/trace/sdk.md:331]: Any thoughts on `(`SpanContext.IsRemote() == true` and `SampledFlag == true`)

or `&&` instead of `and`

?

Comment (Oberon00) [inline specification/trace/sdk.md:331]:
```suggestion
  * Remote parent (`SpanContext.IsRemote() == true`) with `SampledFlag` set
```

Reads cleaner to me.

Comment (arminru) [reply specification/trace/sdk.md:331]: `set` and `not set`? Fine with me as well. @anuraaga WDYT?

Comment (anuraaga) [reply specification/trace/sdk.md:331]: Seems good to me too

Comment (arminru) [reply specification/trace/sdk.md]: adopted https://github.com/open-telemetry/opentelemetry-specification/pull/2220/files#r768534126 instead

Comment (arminru) [reply specification/trace/sdk.md]: adopted https://github.com/open-telemetry/opentelemetry-specification/pull/2220#discussion_r768534126 instead

Comment (arminru) [reply specification/trace/sdk.md]: alright, applied :)

**Review by jmacd — APPROVED** [review]

**Pull request merged**